kernel/Cargo.toml

```diff
@@ -70,5 +70,11 @@ all = ["cvm_guest"]
 
 cvm_guest = ["dep:tdx-guest", "ostd/cvm_guest"]
 
+# Specifying dependencies conditioned on features is not supported.
+# Please see https://github.com/rust-lang/cargo/issues/1197 for more details.
+[target.'cfg(target_arch = "x86_64")'.dependencies.aster-virtio]
+path = "comps/virtio"
+features = ["msix"]
+
```

Comment:
@StevenJiang1110 If msix were a default feature, these lines would be unnecessary. But RISC-V (and loongarch in the future) would need similar things like `default-features = false`. Considering that there is no plan to add msix support for other architectures, is `cfg(not(target_arch = "x86_64"))` ok with you?

Reply:
Indeed, the RISC-V developers have not yet decided whether to enable MSI-X. However, I presume they might opt to enable it; otherwise, they would need to find alternative mechanisms to support device interrupts.

More importantly, since the RISC-V CI has not been introduced, I want to avoid making any changes that might potentially break the RISC-V build.

Therefore, I think using `cfg(target_arch = "loongarch64")` is preferable here.